pkg/builder/action_merge_meta_data_builder.go

```diff
@@ -187,3 +187,120 @@ func (rb *ResizeMergePolicyBuilder) Build() (*proto.ActionMergePolicyDTO, error)
 	mergeSpec.ExecutionTargets = executionTargetList
 	return mergeSpec, nil
 }
+
+
+
+
+// Horizontal Scale Merge Policy DTO builder
+type HorizontalScaleMergePolicyBuilder struct {
+	entityType                *proto.EntityDTO_EntityType
+	aggregationTargets        []*ActionAggregationTargetBuilder
```

Comment:
I don't believe that we need the `aggregationTargets`.

Reply:
Yes, I removed that. Thanks!